pycds/alembic/versions/3d50ec832e47_add_constraint_on_meta_vars.py

```diff
@@ -0,0 +1,57 @@
+"""add constraint on meta vars
+
+Revision ID: 3d50ec832e47
+Revises: 0d99ba90c229
+Create Date: 2022-01-12 13:11:12.480424
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import citext as ci
+from pycds.orm.views.version_84b7fc2596d5 import ObsWithFlags
+
+# revision identifiers, used by Alembic.
+revision = "3d50ec832e47"
+down_revision = "0d99ba90c229"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ### #
+   op.execute("CREATE EXTENSION IF NOT EXISTS citext")
```

Comment:
Interesting. We've not added extensions this way before, but there's no reason it shouldn't be in a migration script.